sdk/servicebus/Azure.Messaging.ServiceBus/src/Resources.resx

```diff
@@ -294,4 +294,7 @@
   <data name="UnsupportedTransportEventType" xml:space="preserve">
     <value>The requested transport event type, '{0}', is not supported by the active transport client.</value>
   </data>
+  <data name="MessageBatchIsLocked" xml:space="preserve">
+    <value>The message batch is currently being used in communication with the Service bus service; messages may not be added until the active operation is complete.</value>
```

Comment:
nit: Service Bus service

Reply:
If I didn't leave a few typos, nobody would believe that I wrote this....